src/templates/en/2019/contributors.html

```diff
@@ -9,7 +9,7 @@
 
 {% block breadcrumb_name_2 %}{{ year }} Contributors{% endblock %}
 
-{% block filter_by_team %}Filter by team: <span id="filtered-contributors">{{ self.contributors() }}</span> contributors{% endblock %}
+{% block filter_by_team %}Filter by team: <span id="filtered-contributors">{{ self.contributors() }}</span> contributors<span id="contributors-total-text" class="hidden"> out of <span id="contributors-total">{{ config.contributors.items() | length }}</span> in total</span>.{% endblock %}
```

Comment:
This sounds verbose to me. How much do you like it? 😁

Reply:
Well obviously I do or I wouldn’t have implemented it 😀

I do like showing the total contributors as much as possible (we do it twice in home screen!) and feel only showing filtered number (especially if coming straight from a link) on this — the contributors page — underplays the total contributors figure.

For some small groups (e.g. translators) it may be obvious for for others (e.g. brainstormers) it could easily be assumed to be the total.

I think we have the space, even on mobile, so why not? Though admittedly in some of the languages it goes to two lines but for some (French) it does that anyway now so still only two lines.

Reply:
Btw you did spot that I don’t show this total when the first number is the total already?

Reply:
I like the idea behind it. I was getting at the verbosity, for example I'd suggest rewording it to something more concise like "46 of 92 contributors"